draft-ietf-mls-architecture.md

```diff
@@ -205,8 +205,6 @@ Clients may wish to do the following:
 
  -  remove one or more members from an existing group;
 
- -  join an existing group;
```

Comment:
I suppose this relates to UserAdd having been removed from the protocol draft?
If so, the idea was to re-introduce it later, so I'm wondering if this requirement shouldn't stay in the architecture draft in the meantime (as a goal for the protocol)

Reply:
I agree with Raphael, this should be kept.